src/mongocxx/test/database.cpp

```diff
@@ -317,7 +316,7 @@ TEST_CASE("Database integration tests", "[database]") {
 
         auto cursor = database.list_collections();
         for (auto&& coll : cursor) {
-            if (coll["name"].get_utf8().value == collection_name) {
+            if (coll["name"].get_utf8().value == stdx::string_view{collection_name}) {
```

Comment:
Not this one, there's no built-in comparison between a `std::string_view` to a `const char*`.  I think this has to stay a string_view.